### ISSUE TITLE
chromedriver helpers - update chromedriver (i.e. download and move to location on PATH)

### ISSUE THREAD

**Issue by stevecondylios** [stevecondylios/RSeleniumHelpers#2]
Some function to automatically

- check that chromedriver is compatible with chrome
- if not, download appropriate chromedriver
- move it to a sensible location (e.g. `/usr/local/bin`) 
- check that the location is on PATH (and potentially give suggestions if it isn't -e.g. edit PATH or add the location to PATH)
- Automatically permissions on MacOS Catalina and Mojave (see [here](https://stackoverflow.com/a/60374958/5783745
))

**Comment by stevecondylios** [comment]
A handy start - some bash to navigate to the current location of chromedriver and to give R permission to access it

```
# https://unix.stackexchange.com/a/602916/302720
"cd $(dirname $(which chromedriver))" # Returns location of application called 'chromedriver'

# https://stackoverflow.com/a/60374958/5783745
"xattr -d com.apple.quarantine chromedriver"


```

**Comment by stevecondylios** [comment]
Implemented in 8e2962cbe01776bbfe1579caf05446d40934be56
